src/Layout/Nav/Nav.js

```diff
@@ -1,25 +1,54 @@
-import React from 'react';
-import {NavLink} from 'react-router-dom';
+import React from "react";
+import { NavLink } from "react-router-dom";
 
 import "./Nav.scss";
 
-const Nav  = ({ setAccessToken }) => {
+const Nav = ({ setAccessToken, loggedUser: { _id}, loggedUsersPokes }) => {
+  const handleLogOut = () => {
+    setAccessToken("");
+  };
 
-    const handleLogOut = () => {
-        setAccessToken("");
-    }
+  const newPokes = () => {
+    const receivedPokes = loggedUsersPokes.filter(
+      (poke) => poke.recipient._id === _id
```

Comment:
Unnecessary parentheses
```suggestion
      poke => poke.recipient._id === _id
```